File: 1392-find-the-difference-of-two-arrays/1392-find-the-difference-of-two-arrays.py
class Solution(object):
    def findDifference(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[List[int]]
        """
        lst1 = []
        for i in nums1:
            if i not in nums2 and i not in lst1:
                lst1.append(i)
        lst2 = []
        for i in nums2:
            if i not in nums1 and i not in lst2:
                lst2.append(i)
        return [lst1,lst2]

File: 1392-find-the-difference-of-two-arrays/1392-find-the-difference-of-two-arrays.java
class Solution {
    public List<List<Integer>> findDifference(int[] nums1, int[] nums2) {
        List<List<Integer>> ans = new ArrayList<>(2);
        List<Integer> arr1 = new ArrayList<>();
        List<Integer> arr2 = new ArrayList<>();
        Set<Integer> hs1 = new HashSet<>();
        for(int i:nums1) hs1.add(i);
        Set<Integer> hs2 = new HashSet<>();
        for(int i:nums2) hs2.add(i);
        for(int i : hs1){
            if(!hs2.contains(i)){
                arr1.add(i);
            }
        }
        for(int i : hs2){
            if(!hs1.contains(i)){
                arr2.add(i);
            }
        }
        ans.add(arr1);ans.add(arr2);
        return ans;
    }
}